pulp/rpm.py

```diff
@@ -0,0 +1,66 @@
+import os
+from functools import wraps
+
+from flask import make_response, request
+from werkzeug.utils import secure_filename
+
+import features
+
+from app import app
+from auth.decorators import process_basic_auth_no_pass
+from auth.registry_jwt_auth import process_registry_jwt_auth
+from endpoints.v2 import require_repo_write
+from endpoints.decorators import (
+    anon_protect,
+    check_pushes_disabled,
+    check_readonly,
+    disallow_for_account_recovery_mode,
+    parse_repository_name,
+    route_show_if,
+)
+from pulp import pulp_bp, pulp_client
+
+# UPLOAD_FOLDER = '/quay-registry'
+# app.config['UPLOAD_FOLDER'] = UPLOAD_FOLDER
+
+
+BASE_PULP_PLUGIN_RPM_ROUTE = "/<repopath:repository>/_pulp/rpm"
+
+
+@pulp_bp.route(BASE_PULP_PLUGIN_RPM_ROUTE, methods=["GET"])
+@parse_repository_name()
+@anon_protect
+def pulp_plugin_support(namespace_name, repo_name):
+    raise NotImplementedError()
+
+
+@pulp_bp.route(BASE_PULP_PLUGIN_RPM_ROUTE, methods=["POST"])
+@route_show_if(features.PULP_PLUGIN)
```

Comment:
`PULP_PLUGIN` needs to be added to features

Reply:
can we add pulp tu the generic plugin framework? We have another plugin for model registry that needs to be added as well